fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.bulkimport;
+
+import java.util.HashMap;
+import java.util.List;
+
+import javax.batch.api.listener.JobListener;
+import javax.batch.runtime.context.JobContext;
+import javax.inject.Inject;
+
+public class ImportJobListener implements JobListener {
+    @Inject
+    JobContext jobContext;
+
+    private long jobStartTimeInMS, jobEndTimeInMS;
+
+    public ImportJobListener() {
+
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void afterJob() {
```

Comment:
Maybe you can separe this method in smalls ones.
In this way you make the ciclomatic complexiti donw to.

Reply:
I don't really think currently it's complicated enough to have to be separated into multiple, and there is no common codes used in it. but I do plan to separate the future bulkdata response generation and metric generation from this function into 2 different functions. and this will be done in issue #671